sass/directives/_carousels.scss

```diff
@@ -88,6 +88,63 @@
   }
 }
 
+@mixin resource-header {
+  h2, p, h3, h1 {
+    display: inline-block;
+  }
+
+  nav {
+    display: inline-block;
+    float: right;
+  }
+}
+
+@mixin resource-carousel-arrow {
+  z-index: z($z-context, page_content, $page_content, carousel__arrow);
+  color: $inactive-carousel-button-color;
+  font-size: $direction-arrow-size / 2;
+  cursor: pointer;
+  border: 1px solid $inactive-carousel-button-color;
+  padding: $base-padding / 2;
+
+  &:hover,
+  &:active,
+  &:focus {
+    color: $active-carousel-button-color;
+    border: 1px solid $active-carousel-button-color;
+  }
+
+  &--left {
+    margin-right: 1px;
+  }
+}
+
+@mixin resource-continue-block {
```

Comment:
to make sure there's always the right amount of space at the bottom of the block, add padding-bottom to the resource-block that is equal to the height of the continue block

you can remove the margin-top on the continue block. the content area has padding already that will take care of that

but right now, the content area potentially bleeds into the continue area (shown here in pink b/c github is not uploading images right now: https://screencast.com/t/XfIT8mwVp4AO)